src/ui/server/helpers/http/index.ts

```diff
@@ -0,0 +1,18 @@
+import * as dotenv from 'dotenv';
+import express from 'express';
+import { SERVICE_NAME } from '../../constants';
+
+dotenv.config();
+const { PORT } = process.env;
+
+/**
+ * Sets up error handling and listening for an Express server.
+ * @param server - An instance of the Express server.
+ * @returns void
+ */
+
+// @ts-ignore
```

Comment:
Not a blocker - i'm sure there's a way to pass express server as types. I'll pick this up 👍

Reply:
Thanks, I was unable to find it, was hoping to speak with you regarding this anyways.

Reply:
Will have a look 😊 When I see `ts-ignore` I go into investigation mode